types/object-hash/object-hash-tests.ts

```diff
@@ -16,6 +16,7 @@ hash(obj); // $ExpectType string
 // @ts-expect-error
 hash(undefined);
 hash(''); // $ExpectType string
+hash([]); // $ExpectType string
```

Comment:
please copy tests from this file https://github.com/DefinitelyTyped/DefinitelyTyped/pull/63410/files#diff-ad46eba89b7ffb5011e243455996794dcb932f326a5721712a782620a4c5c150